bokehjs/src/lib/models/annotations/box_annotation.ts

```diff
@@ -111,6 +119,25 @@ export class BoxAnnotationView extends AnnotationView {
 
     ctx.restore()
   }
+
+  bbox_contains(sx: number, sy: number): boolean {
+    if (this.model.in_cursor == null)
+      return false
+    const bbox = new BBox({x0: this.sleft-3, y0: this.stop-3, x1: this.sright+3, y1: this.sbottom+3})
+    return bbox.contains(sx, sy)
+  }
+
+  cursor(sx: number, sy: number): string | null {
+    const tol = 3
```

Comment:
I presume this number and the above are the same. This has to be extracted. Is this somehow related to border width (line width)? If so, lets use that instead.

Reply:
It's just to make the UI landing area slightly bigger for users, `line_width+1` is probably reasonable.

Reply:
fixed in 2cb8e88